src/screens/LoginView/LoginView.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import {Text, ScrollView, View} from 'react-native';
+import TextBoxWithButton from '../../components/TextBoxWithButton/TextBoxWithButton';
+import TextBox from '../../components/TextBox/TextBox';
+import CustomButton from '../../components/CustomButton/CustomButton';
+import {styles} from '../RegisterView/RegisterViewStyle';
+
+const LoginView = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  return(
+    <ScrollView style={styles.container}>
+      <Text style={styles.title}>Sign In</Text>
+      <View style={{marginBottom: 10}}>
+        <Text style={styles.textInputLabel}>Email</Text>
+        <TextBox style={styles.textInputLabel} value={email} onChange={setEmail} />
+      </View>
+      <View>
+        <Text style={styles.textInputLabel}>Password</Text>
+        <TextBoxWithButton style={styles.textInputLabel} type='password' value={password} onChange={setPassword}  />
+      </View>
+      <View style={{marginTop: 30, marginBottom: 10}}>
```

Comment:
Do not add styles directly, it is better to import them from a styles file.

Reply:
Done.